### ISSUE TITLE
Error of make test on release-1.4 branch

### ISSUE THREAD

**Issue by jianzhangbjz** [kubernetes/kubernetes#38596]
Hey Guys,

When running `make test` command on release-1.4 branch, I got the below errors, my GO version is `go version go1.7.1 linux/amd64`, how to fix it or something else have I missed?
```
--- FAIL: TestDeletePods (0.10s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x0]

goroutine 78 [running]:
panic(0x1ad9860, 0xc420012130)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
testing.tRunner.func1(0xc42025b080)
	/usr/local/go/src/testing/testing.go:579 +0x25d
panic(0x1ad9860, 0xc420012130)
	/usr/local/go/src/runtime/panic.go:458 +0x243
k8s.io/kubernetes/pkg/kubectl/cmd/util.PrintSuccess(0xc420043f80, 0x93a21d, 0xc42048f800, 0xc420293c20, 0x0, 0x1cbada3, 0x3, 0xc42048f8a0, 0x4, 0x1cbebc1, ...)
	/home/zhangjian/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/util/printing.go:60 +0x66
k8s.io/kubernetes/pkg/kubectl/cmd.(*DrainOptions).waitForDelete.func1(0x1a2f260, 0xc420293c80, 0xc420311be7)
	/home/zhangjian/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/drain.go:419 +0x241
k8s.io/kubernetes/pkg/util/wait.WaitFor(0xc420284740, 0xc420284720, 0xc420293c20, 0x0, 0x0)
	/home/zhangjian/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/util/wait/wait.go:210 +0x70
k8s.io/kubernetes/pkg/util/wait.pollInternal(0xc420284740, 0xc420284720, 0x0, 0x0)
	/home/zhangjian/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/util/wait/wait.go:165 +0x99
k8s.io/kubernetes/pkg/util/wait.pollImmediateInternal(0xc420284740, 0xc420284720, 0xc420284740, 0xc420284720)
	/home/zhangjian/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/util/wait/wait.go:182 +0x5a
k8s.io/kubernetes/pkg/util/wait.PollImmediate(0x5f5e100, 0x2540be400, 0xc420284720, 0x0, 0x0)
	/home/zhangjian/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/util/wait/wait.go:171 +0x4d
k8s.io/kubernetes/pkg/kubectl/cmd.(*DrainOptions).waitForDelete(0xc420311e38, 0xc42015aa00, 0x8, 0x8, 0x5f5e100, 0x2540be400, 0xc42048f870, 0xc4202a5770, 0xc4202a5760, 0xc4202a5750, ...)
	/home/zhangjian/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/drain.go:433 +0xf1
k8s.io/kubernetes/pkg/kubectl/cmd.TestDeletePods(0xc42025b080)
	/home/zhangjian/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/drain_test.go:624 +0x255
testing.tRunner(0xc42025b080, 0x1dc1428)
	/usr/local/go/src/testing/testing.go:610 +0x81
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:646 +0x2ec
FAIL	k8s.io/kubernetes/pkg/kubectl/cmd	0.270s
``` 
```
--- FAIL: TestCloneTLSConfig (0.00s)
	http_test.go:72: New fields seen in http.Transport: [DynamicRecordSizingDisabled Renegotiation]
		Add to CopyClientTLSConfig if client-relevant, then add to expected list in TestCopyClientTLSConfig
FAIL
FAIL	k8s.io/kubernetes/pkg/util/net	0.056s
```

**Comment by xiangpengzhao** [comment]
dup of #33936. fixed by #39213.

we can close this.

/sig api-machinery
@thockin

**Comment by jianzhangbjz** [comment]
Sure, close it.
